packages/dialtone-vue3/components/popover/popover.vue

```diff
@@ -507,6 +512,14 @@ export default {
             (appendTo instanceof HTMLElement);
       },
     },
+
+    /**
+     * Set this prop to true and popover component will support hovercard behaviour
```

Comment:
explain specifically what "hovercard behaviour" means. In this case it causes it to trigger mouseenter/mouseleave events on a timer delay of 300ms.

Reply:
K, updated to:
```
/**
     * Set this prop to true and popover component will support hovercard behaviour
     * It will open on mouseenter and close on mouseleave with timer delay of 300ms
     */
```